docs/about/release-notes.rst

```diff
@@ -54,7 +54,7 @@ Breaking changes
 Bugfixes
 ~~~~~~~~
 
-* Fix an issue in the setup of automatic bin edges when giving a bin count to :py:func:`scipp.bin`, :py:func:`scipp.hist`, and :py:func:`scipp.rebin`. This lead to a miniscule shift in the resulting edges, which is irrelevant in most cases `#2838 <https://github.com/scipp/scipp/pull/2838>`_.
+* Fix an issue in the setup of automatic bin edges when giving a bin count to :py:func:`scipp.bin`, :py:func:`scipp.hist`, and :py:func:`scipp.rebin`. This lead to a miniscule shift in the resulting edges, which is irrelevant in most cases, but results in events at the lower bin bound to be placed in a correct but lower bin than expected `#2838 <https://github.com/scipp/scipp/pull/2838>`_.
```

Comment:
```suggestion
* Fix an issue in the setup of automatic bin edges when giving a bin count to :py:func:`scipp.bin`, :py:func:`scipp.hist`, and :py:func:`scipp.rebin`. This lead to a miniscule shift in the resulting edges, which is irrelevant in most cases, but results in events at the lower bin bound to be placed in a lower bin than expected `#2838 <https://github.com/scipp/scipp/pull/2838>`_.
```

Reply:
Why? It is not in the wrong bin, the edges are wrong.

Reply:
hm, subtle difference, but ok. Go ahead with merging